communication/templates/lobby_create.html

```diff
@@ -97,7 +97,8 @@ <h1>Scribble.rs</h1>
     <div id="join-lobby" class="tab-content">
         <div class="join-lobby-data">
             <div class="table-wrapper-wrapper">
-                <button id="refresh-button" onclick="loadLobbyTable()">Refresh</button>
+	            <button id="quick-join-button" onclick="quickJoinLobby()">Quick Join</button>
```

Comment:
I'm not that great with Go templates but if someone could add a feature where if the `lobbyLength` array is empty, then disabled this button.

Reply:
I guess it'd be fine for this to always be enabled and dynamically react depending on whether there's a suitable lobby or not.